storage/src/adb/keyless.rs

```diff
@@ -328,6 +332,24 @@ impl<E: Storage + Clock + Metrics, V: Codec, H: CHasher> Keyless<E, V, H> {
         Ok(loc)
     }
 
+    /// Get the metadata associated with the last commit, or None if no commit has been made or
+    /// there is no metadata associated with the last commit.
+    pub async fn get_metadata(&self) -> Result<Option<V>, Error> {
```

Comment:
nit: `Option<(u64, V)>`

Reply:
I originally suggested for the other dbs but noted your comment about how it wasn't supported to fetch by position.